.github/workflows/generate_intel_image.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         include:
-          - base-image: intel/oneapi-basekit:2025.2.0-0-devel-ubuntu22.04
+          - base-image: intel/oneapi-basekit:2025.3.0-0-devel-ubuntu24.04
```

Comment:
I have been using this in my local builds for a while, so probably Intel is fine at least